contributor_docs/hi/contributor_guidelines.md

```diff
@@ -0,0 +1,528 @@
+# योगदानकर्ता दिशानिर्देश
+
+योगदानकर्ता दिशानिर्देशों में आपका स्वागत है! यह दस्तावेज़ उन नए योगदानकर्ताओं के लिए है जो p5.js में कोड का योगदान करना चाहते हैं, ऐसे योगदानकर्ता जो कुछ तकनीकी चरणों पर अपनी यादें ताज़ा करना चाहते हैं, या p5.js में कोड योगदान के साथ कुछ और करना चाहते हैं।
+
+यदि आप p5.js रिपॉजिटरी के बाहर योगदान करना चाहते हैं (ट्यूटोरियल लिखना, कक्षाओं की योजना बनाना, कार्यक्रमों का आयोजन करना), तो कृपया इसके बजाय अन्य प्रासंगिक पृष्ठों पर एक नज़र डालें। प्रबंधकों या अनुरक्षकों को समस्याएँ की समीक्षा करने और अनुरोधों को खींचने के संबंध में [कार्यपालिका दिशानिर्देश](https://github.com/processing/p5.js/blob/main/contributor_docs/steward_guidelines.md) अधिक उपयोगी लग सकते हैं।
+
+यह अपेक्षाकृत लंबा और व्यापक दस्तावेज़ है लेकिन हम सभी चरणों और बिंदुओं को यथासंभव स्पष्ट रूप से इंगित करने का प्रयास करेंगे। अपने लिए प्रासंगिक अनुभाग ढूंढने के लिए विषय-सूची का उपयोग करें। यदि अनुभाग आपके नियोजित योगदान के लिए प्रासंगिक नहीं हैं, तो बेझिझक उन्हें छोड़ दें।
+
+**यदि आप एक नए योगदानकर्ता हैं, तो आप पहले खंड, "सभी समस्याएँ के बारे में" से शुरुआत करना चाहेंगे। यदि आप विकास प्रक्रिया का चरण-दर-चरण सेटअप चाहते हैं, तो आप "डेवलपर्स के लिए त्वरित शुरुआत करें" अनुभाग देख सकते हैं.**
+
+# Table of Contents
+
+- [सभी समस्याओं के बारे में](#समस्याएँ-के-बारे-में-सब-कुछ)
+  - [समस्याएँ क्या हैं?](#समस्याएँ-क्या-हैं?)
+  - [समस्या खाका](#समस्या-खाका)
+    - [कोई बग मिला](#बग-मिला)
+    - [मौजूदा सुविधा में सुधार](#मौजूदा-सुविधा-में-सुधार)
+    - [नई सुविधा का अनुरोध](#नई-सुविधा-का-अनुरोध)
+    - [चर्चा](#चर्चा)
+- [p5.js कोडबेस पर काम कर रहे हैं](#p5.js-कोडबेस-पर-काम-कर-रहे-हैं)
+  - [डेवलपर्स के लिए त्वरित शुरुआत](#डेवलपर्स-के-लिए-त्वरित-शुरुआत)
+  - [गिटहब संपादन कार्यक्षमता का उपयोग](#गिटहब-संपादन-कार्यक्षमता-का-उपयोग)
+  - [p5.js को फॉर्क करें और अपने फॉर्क से काम करें](#p5.js-को-फॉर्क-करें-और-अपने-फॉर्क-से-काम-करें)
+    - [गिटहब डेस्कटॉप का उपयोग](#गिटहब-डेस्कटॉप-का-उपयोग)
+    - [गिट कमांड लाइन इंटरफ़ेस का उपयोग](#गिट-कमांड-लाइन-इंटरफ़ेस-का-उपयोग)
+  - [कोडबेस विश्लेषण](#कोडबेस-विश्लेषण)
+  - [निर्माण सेटअप](#निर्माण-सेटअप)
+  - [गिट वर्कफ़्लो](#गिट-वर्कफ़्लो)
+    - [सोर्स कोड](#सोर्स-कोड)
+    - [यूनिट टेस्ट](#यूनिट-टेस्ट)
+    - [इनलाइन दस्तावेज़ीकरण](#इनलाइन-दस्तावेज़ीकरण)
+    - [अंतरराष्ट्रीयकरण](#अंतरराष्ट्रीयकरण)
+    - [पहुँचनीयता](#पहुँचनीयता)
+  - [कोड मानक](#कोड-मानक)
+  - [सॉफ़्टवेयर डिज़ाइन सिद्धांत](#सॉफ़्टवेयर-डिज़ाइन-सिद्धांत)
+- [पुल रिक्वेस्ट](#पुल-रिक्वेस्ट)
+  - [पुल रिक्वेस्ट बनाना](#पुल-रिक्वेस्ट-बनाना)
+    - [पुल रिक्वेस्ट के बारे में जानकारी](#पुल-रिक्वेस्ट-के-बारे-में-जानकारी)
+    - [शीर्षक](#शीर्षक)
+    - [हल](#हल)
+    - [परिवर्तन](#परिवर्तन)
+    - [परिवर्तन के स्क्रीनशॉट](#परिवर्तन-के-स्क्रीनशॉट)
+    - [पीआर-चेकलिस्ट](#पीआर-चेकलिस्ट)
+    - [रिबेस और रेसोल्वे कनफ्लिक्ट](#रिबेस-और-रेसोल्वे-कनफ्लिक्ट)
+  - [चर्चा करें और संशोधन करें](#चर्चा-करें-और-संशोधन-करें)
+
```

Comment:
Hi, have you checked all the links of table of contents if they are working?

Reply:
Yeah these links are working fine.